modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -47,11 +47,11 @@ module.exports = ({
         Modules: path.resolve(modulesDir)
       },
       modules: [
-        // TODO change this if we decide to namespace the
-        // apostrophe module itself
+        'node_modules',
         `${apos.npmRootDir}/node_modules/apostrophe/node_modules`,
         `${apos.npmRootDir}/node_modules`
-      ]
+      ],
+      symlinks: false
```

Comment:
Also needed to ensure symlinks are not resolved to absolute paths, which breaks the normal recursive resolution algorithm.